src/Microsoft.AspNetCore.ReactServices/npm/redux-typed/src/StrongActions.ts

```diff
@@ -38,5 +38,5 @@ export interface Reducer<TState> extends Function {
 }
 
 export interface ActionCreatorGeneric<TState> extends Function {
-    (dispatch: Dispatch, getState: () => TState): any;
+    (dispatch: Dispatch<TState>, getState: () => TState): any;
```

Comment:
Nice catch! Need this for my issue with TS2.0 PR & React as well.
`node_modules/redux-typed/StrongActions.d.ts (16,16): error TS2314: Generic type 'Dispatch<S>' requires 1 type argument(s).`